code/game/objects/items/devices/flashlight.dm

```diff
@@ -254,11 +254,13 @@
 	var/fuel = 0
 	var/on_damage = 7
 	var/produce_heat = 1500
+	var/frng_min = 800

```

Comment:
Mark these with // HAVEN comments, so we know what to keep in case of a conflict.